boards/others/neorv32/doc/index.rst

```diff
@@ -37,9 +37,10 @@ CPU
 The default board configuration assumes the NEORV32 CPU implementation has the
 following RISC-V ISA extensions enabled:
 
-- C (Compresses Instructions)
+- I (Base Integer Instruction Set, 32-bit)
```

Comment:
Is C no longer used here by default? How does it affect rom size?

Reply:
It is not. I have updated the default board configuration to match the https://github.com/stnolting/neorv32/blob/main/rtl/processor_templates/neorv32_ProcessorTop_UP5KDemo.vhd for now, which only enables RV32I and M.

In a follow-up PR, I will introduce board variants, each matching one of the pre-defined NEORV32 processor templates available [upstream](https://github.com/stnolting/neorv32/tree/main/rtl/processor_templates) (MinimalBoot and UP5KDemo for sure, not sure Minimal is very interesting, as it lacks UART0 for the console).